mm/slab.c

```diff
@@ -3231,7 +3231,7 @@ slab_alloc_node(struct kmem_cache *cachep, gfp_t flags, int nodeid, size_t orig_
 
 	ptr = kfence_alloc(cachep, orig_size, flags);
 	if (ptr)
-		goto leave;
+		goto out_hooks;
```

Comment:
But `out_hooks` and `out` themselves are inconsistent?

Reply:
There is already an 'out' in this function which we cannot jump to.

Reply:
Ah, haven't noticed it.